src/tracks/playabletrack/wavetrack/ui/WaveTrackAffordanceControls.cpp

```diff
@@ -613,7 +613,9 @@ void WaveTrackAffordanceControls::StartEditSelectedClipSpeed(
 void WaveTrackAffordanceControls::OnRenderClipStretching(
    AudacityProject& project)
 {
-   auto [track, it] = SelectedIntervalOfFocusedTrack(project);
+   const auto p = SelectedIntervalOfFocusedTrack(project);
+   auto track = p.first;
+   auto it = p.second;
```

Comment:
The compile on macOS doesn't let you capture a structured binding name directly in a lambda.
Another fix is to change the lambda capture below from `[&]` to `[track = track, interval = interval]`

Reply:
Trying ... 🤞